packages/web-pkg/src/components/Search/ResourcePreview.vue

```diff
@@ -89,9 +79,22 @@ export default defineComponent({
       eventBus.publish('app.search.options-drop.hide')
     }
 
+    const additionalAttrs = computed(() => {
+      if (!props.isClickable) {
+        return {}
+      }
+
+      return {
+        isResourceClickable: true,
+        parentFolderLink: getParentFolderLink(unref(resource)),
+        onClick: resourceClicked,
```

Comment:
does this work? i think `v-bind` is only for attributes and `v-on` is for events

Reply:
It works